library/Ivoz/Provider/Domain/Model/QueueMember/QueueMemberRepository.php

```diff
@@ -7,4 +7,18 @@
 
 interface QueueMemberRepository extends ObjectRepository, Selectable
 {
+    /**
+     * @param int $userId
+     * @return array<QueueMemberInterface>
+     */
+    public function findByUserId(int $userId): array;
```

Comment:
Add tests in schema/tests

Reply:
Added it_finds_by_user_id and it_finds_one_by_queue_and_extension repository tests